### PR TITLE
README updated and matplotlib dependency made optional

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -26,14 +42,16 @@ normally used in B physics is also provided.
...
-* `MIT <http://opensource.org/licenses/MIT>`_ license (open source)
-* `Documentation <http://probfit.readthedocs.org/>`_
+*`Documentation <http://probfit.readthedocs.org/>`_
 * The tutorial is an IPython notebook that you can view online
   `here <http://nbviewer.ipython.org/urls/raw.github.com/scikit-hep/probfit/master/tutorial/tutorial.ipynb>`_.
   To run it locally: `cd tutorial; ipython notebook --pylab=inline tutorial.ipynb`.
-* Dependencies:
-   - `iminuit <http://iminuit.readthedocs.org/>`_
-   - `numpy <http://www.numpy.org/>`_
-   - `matplotlib <http://matplotlib.org/>`_ (optional, for plotting)
```
Comment (henryiii) [inline README.rst]: I believe this is currently *not* optional - it really should be made optional, though!

Comment (HDembinski) [reply README.rst]: Agreed.

Comment (eduardo-rodrigues) [reply README.rst]: Am working on that. Will commit soon …

(Aside, @HDembinski: kindly release the tagged 1.1.0 version I prepared yesterday. It addresses wishes by several people ;-). This is now work that is 2nd priority. Thanks a lot.)

**Comment by eduardo-rodrigues** [comment]
Hi @HDembinski and @henryiii, I believe this is now ready to go. Thanks for the checks.

File: README.rst [hunk 2]
```diff
@@ -26,14 +46,16 @@ normally used in B physics is also provided.
     minuit.migrad()
     unbinned_likelihood.draw(minuit)
 
+Documentation and Tutorial
+--------------------------
 
-* `MIT <http://opensource.org/licenses/MIT>`_ license (open source)
-* `Documentation <http://probfit.readthedocs.org/>`_
+*`Documentation <http://probfit.readthedocs.org/>`_
```
Comment (henryiii) [inline README.rst]: A space is now missing here, it won't render correctly.

Comment (eduardo-rodrigues) [reply README.rst]: Good points, thanks!

File: README.rst [hunk 3]
```diff
@@ -1,19 +1,35 @@
 .. -*- mode: rst -*-
 
-.. image:: https://travis-ci.org/scikit-hep/probfit.png?branch=master
-   :target: https://travis-ci.org/scikit-hep/probfit
+probfit
+=======
 
+.. image:: https://img.shields.io/pypi/v/probfit.svg
```
[the remark this answers is not in the document]
Comment (HDembinski) [reply README.rst]: Ok, nevermind, the shield badge is for pypi, so there is no replacement.

[the remark this answers is not in the document]
Comment (eduardo-rodrigues) [reply README.rst]: Yep, no alternative here.

**Comment by HDembinski** [comment]
@eduardo-rodrigues If you want to merge this, please go ahead.

**Comment by eduardo-rodrigues** [comment]
Any of us can do it :). OK, doing it … Thanks both for the good eyes and checks.

**Pull request merged**